FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -350,11 +333,7 @@ namespace edm {
       // <site-local-config>
       // <site name="FNAL">
       //   <subsite name="FNAL_SUBSITE"/>
-      //   <event-data>
-      //     <catalog url="trivialcatalog_file:/x/y/z.xml"/>
-      //   </event-data>
```

Comment:
It could be good to add an example of how the corresponding Rucio catalog entry would look like, e.g.
```xml
TODO
```